### PR TITLE
feat(workflow): Show assignee/status in Similar Issues

### PR DESCRIPTION
**Pull request by billyvg**
image

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/groupSimilar/similarItem.jsx [hunk 1]
```diff
@@ -121,6 +121,8 @@ const SimilarIssueItem = React.createClass({
                 orgId={orgId}
                 projectId={projectId}
                 lastSeen={null}
+                showAssignee
+                showStatus
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/groupSimilar/similarItem.jsx]: do you need to pass in assignedTo here? or it's coming off issue?

**Review by MaxBittker — APPROVED** [review]
lgtm 👍

**Comment by billyvg** [comment]
it's in `data` (which in this case is an issue)

**Comment by benvinegar** [comment]
Why not do the same avatar UI we use in the regular list of issues? cc @ckj

**Comment by benvinegar** [comment]
Similarly, I'd expect resolved/unresolved to be reflected in the "line-through" design we have on the issues page.

**Comment by billyvg** [comment]
@benvinegar This is on the similar issues view, I think adding the avatar UI will add clutter especially when that particular view is supposed to show you other issues that should be grouped.

I think we should do `line-through` for resolved status though.

**Pull request merged**